app/controllers/admin/getting_started_guide_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Admin::GettingStartedGuideController < ApplicationController
+
+  before_filter :ensure_is_admin
+
+  rescue_from ReactOnRails::PrerenderError do |err|
+    Rails.logger.error(err.message)
+    Rails.logger.error(err.backtrace.join("\n"))
+    redirect_to root_path, flash: { error: "Error prerendering in react_on_rails. See server logs." }
+  end
+
+  def index
+    render locals: { props: data }
+  end
+
+  private
+
+  def data
+    path_parts = request.env['PATH_INFO'].split("/getting_started_guide")
+    has_deep_path = !(path_parts.count == 1 || path_parts == "")
+    path_string = has_deep_path ? path_parts[1] : "";
```

Comment:
Hmm. this part comes from my branch (and it seems I have refactored an important part away - I had originally root path of component marked as "/", but it caused unnecessary history update.)
I'll change my code to be:
```
    has_sub_path = (path_parts.count == 2 && path_parts[1] != "/")
    sub_path = has_sub_path ? path_parts[1] : "";
```